bench/app.py

```diff
@@ -96,28 +96,29 @@ def remove_from_excluded_apps_txt(app, bench_path='.'):
 
 def get_app(git_url, branch=None, bench_path='.', skip_assets=False, verbose=False,
 		postprocess=True, overwrite=False):
```

Comment:
Does what https://github.com/frappe/bench/pull/913 was meant to do. Here `get_app` works only for local git repositories